lib/hanami/cli/generators/app/operation.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "erb"
+require "dry/files"
+require_relative "../../errors"
+
+module Hanami
+  module CLI
+    module Generators
+      module App
+        # @since x.x.x
+        # @api private
+        class Operation
+          # @since x.x.x
+          # @api private
+          def initialize(fs:, inflector:)
+            @fs = fs
+            @inflector = inflector
+          end
+
+          # @since x.x.x
+          # @api private
+          def call(app, key, slice)
+            context = OperationContext.new(inflector, app, slice, key)
+
+            if slice
+              generate_for_slice(context, slice)
+            else
+              generate_for_app(context)
+            end
+          end
+
+          private
+
+          attr_reader :fs
+
+          attr_reader :inflector
+
+          def generate_for_slice(context, slice)
+            slice_directory = fs.join("slices", slice)
+            raise MissingSliceError.new(slice) unless fs.directory?(slice_directory)
+
+            if context.namespaces.any?
+              fs.mkdir(directory = fs.join(slice_directory, "operations", context.namespaces))
```

Comment:
I don't think we should hardcode `"operations"` in the directory path here. Operations should be able to be created _anywhere_ within an app or slice.

(This comment also applies to all the other places we've hard-coded "operations" within this PR)